src/lib/components/ui/ScrollView/ScrollView.scss

```diff
@@ -0,0 +1,193 @@
+@import '../../../styles/tools/caret';
+@import '../../../styles/tools/hide-scrollbar';
+@import '../../../styles/tools/reset';
+@import '../../../styles/tools/transitions';
+@import 'theme';
+
+// 1. Scrolling shadows are implemented as pseudo elements. This way we can customise them only
+//    thanks to custom properties.
+// 2. Clip scrolling shadows.
+// 3. Create stacking context just to avoid scrolling shadows being covered by content.
+// 4. Make the content bounding rectangle spread beyond the visible part.
+// 5. Prevent undesired vertical scrolling that may occur with tables inside.
+
+$_arrow-inner-offset: 0.5rem;
+$_arrow-outer-offset: 1rem;
+
+.root {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  overflow: hidden; // 2.
+
+  // 1.
+  &::before,
+  &::after {
+    @include transition((visibility, opacity, transform));
```

Comment:
Is there any reason to have it wrapped with two brackets? It is same for followings lines.

Reply:
It's a list, see https://github.com/react-ui-org/react-ui/blob/master/src/lib/styles/tools/_transitions.scss.